Resources/Prototypes/Entities/Constructible/Ground/wires.yml

```diff
@@ -95,8 +95,13 @@
   parent: ApcExtensionCable
   components:
   - type: NodeContainer
-    nodes:
-    - !type:AdjacentNode
-      nodeGroupID: HVPower
-    - !type:AdjacentNode
-      nodeGroupID: Apc
+    nodeTypes: { HVPower : ["AdjacentNode"], Apc : ["AdjacentNode"] }
+
+- type: entity
+  id: Generator
+  name: Depriciated Generator
+  parent: DebugGenerator
+  components:
+  - type: PowerSupplier
+    voltage: High
+    supplyRate: 100000
```

Comment:
This should be removed as well, as all the other depriciated power prototypes are gone too.

Reply:
gaaah, it was automatically merged like that